core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/ai-image-prompt.component.html

```diff
@@ -4,28 +4,35 @@
         [dismissableMask]="true"
         [draggable]="false"
         [resizable]="false"
-        [style]="{ width: '800px' }"
+        [style]="{ width: '880px' }"
         (onHide)="hideDialog()"
         appendTo="body"
         header="{{ 'block-editor.extension.ai-image.dialog-title' | dm }}">
-        <div class="dialog-prompts__wrapper grid">
-            <dot-ai-image-prompt-input
-                class="col"
-                [isLoading]="vm.status === ComponentStatus.LOADING"
-                [selected]="vm.selectedPromptType === 'input'"
-                (click)="selectType('input', vm.selectedPromptType)"
-                (promptChanged)="generateImage($event)"
-                placeholder="{{ 'block-editor.extension.ai-image.input-text.placeholder' | dm }}"
-                type="input" />
-
-            <dot-ai-image-prompt-input
-                class="col"
-                [isLoading]="vm.status === ComponentStatus.LOADING"
-                [selected]="vm.selectedPromptType === 'auto'"
-                (click)="selectType('auto', vm.selectedPromptType)"
-                (promptChanged)="generateImage($event)"
-                placeholder="{{ 'block-editor.extension.ai-image.auto-text.placeholder' | dm }}"
-                type="auto" />
+        <div class="dialog-prompt__wrapper flex">
+            <dot-ai-image-prompt-form
+                *ngIf="vm.showDialog"
+                [generatedValue]="vm.images[vm.galleryActiveIndex]"
+                [isLoading]="vm.isLoading"
+                (value)="generateImage($event)"
+                (orientation)="vm.orientation = $event" />
+            <div [class.dialog-prompt_gallery]="vm.images.length || vm.isLoading">
+                <dot-ai-image-prompt-gallery
+                    [isLoading]="vm.isLoading"
+                    [images]="vm.images"
+                    [index]="vm.galleryActiveIndex"
```

Comment:
Instead of the index, something more meaningful like a chosen image/selected example.

Reply:
agree changed to `activeImageIndex`